src/neo/IO/Caching/DataCache.cs

```diff
@@ -144,19 +145,14 @@ public void Delete(TKey key)
 
         protected abstract void DeleteInternal(TKey key);
 
-        /// <summary>
-        /// Find the entries that start with the `key_prefix`
-        /// </summary>
-        /// <param name="key_prefix">Must maintain the deserialized format of TKey</param>
-        /// <returns>Entries found with the desired prefix</returns>
-        public IEnumerable<(TKey Key, TValue Value)> Find(byte[] key_prefix = null)
+        public IEnumerable<(TKey key, TValue value)> Seek(byte[] key)
```

Comment:
Add `direction`? Seek forward/backward.

Reply:
OK, I'll add it. Both leveldb and rocksdb support seek previous.

Reply:
Can you create another PR to add `Seek()`?

Reply:
see https://github.com/neo-project/neo/pull/1740